tests/integration/standard/test_shard_aware.py

```diff
@@ -37,8 +38,8 @@
 
 
 def setup_module():
-    os.environ['SCYLLA_EXT_OPTS'] = "--smp 4 --memory 2048M"
-    use_cluster('shared_aware', [3], start=True)
+    os.environ['SCYLLA_EXT_OPTS'] = "--smp 2"
```

Comment:
I understand you want to fit the test in GitHub action, but changing the number of shards changes the test logic.

The test is querying specific keys, that should make the query go to specific shard, when smp changes we need to pick new keys, or the target shard.

Reply:
I know I've written them initially

Anyhow we are using exactly the tracing for confirmation, 

I would remove the assert before figuring out what changed in Scylla that we can't trust them anymore.

It might hide a bug in the driver code.

Also I was quite sure the java tests had them